sparse/coo/__init__.py

```diff
@@ -5,6 +5,8 @@
                      eye, full, full_like, zeros, zeros_like, ones, ones_like,
                      kron, argwhere, isposinf, isneginf)
 
+from numpy.core._multiarray_umath import result_type
```

Comment:
Beware, this is likely to break in a future NumPy release — this isn’t a public API